settings.gradle.kts

```diff
@@ -53,8 +53,12 @@ dependencyResolutionManagement {
     }
 }
 
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version("0.8.0")
```

Comment:
Why is it needed?

Reply:
It's an official Gradle plugin that will automatically discover JDKs requested by Gradle builds, or download them if missing.


https://github.com/gradle/foojay-toolchains